vignettes/opart_poisson.Rmd

```diff
@@ -155,45 +155,116 @@ str(sfit)
 ```
```

Comment:
Hi @tdhock ,

Can you review these changes ? I won't be able to meet you today as I have to move to my new residence. Let me know if we can meet tomorrow.